charts/kubevault-metrics/templates/vaultserver/ops-kubevault-com-vaultopsrequest.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: metrics.appscode.com/v1alpha1
+kind: MetricsConfiguration
+metadata:
+  name: ops-kubevault-com-vaultopsrequest
+spec:
+  targetRef:
+    apiVersion: ops.kubevault.com/v1alpha1
+    kind: VaultOpsRequest
+  commonLabels:
+    - key: app
+      valuePath: .spec.vaultRef.name
+    - key: app_namespace
+      valuePath: .metadata.namespace
+    - key: type
+      valuePath: .spec.type
+  metrics:
+    - name: type
```

Comment:
type is already added as common label, so another metrics named type seems redundant.

Reply:
Yes. Kinda redundent. This metrics is used only to check type of the ops request. We don't use the metrics in alerts or dashboard. We only use status metrics.

Reply:
Removed